src/main/java/org/skriptlang/skript/bukkit/toolcomponent/elements/EffSecCreateToolRule.java

```diff
@@ -0,0 +1,111 @@
+package org.skriptlang.skript.bukkit.toolcomponent.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.Section;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.Trigger;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.registrations.EventValues;
+import ch.njol.skript.util.Getter;
+import ch.njol.skript.variables.Variables;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.meta.components.ToolComponent.ToolRule;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.toolcomponent.ToolRuleWrapper;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+@Name("New Tool Rule")
+@Description("Creates a new tool rule.")
+@Examples({
+	"create a new tool rule and store it in {_toolrule}:",
+		"\tset the tool rule blocks to oak log, stone and obsidian",
+		"\tset the tool rule speed to 5",
+		"\tenable the tool rule drops",
+	"add {_toolrule} to the tool rules of {_item}"
+})
+@RequiredPlugins("Minecraft 1.20.6+")
+@Since("INSERT VERSION")
+public class EffSecCreateToolRule extends Section {
+	// TODO: Change to secpression when merged
+	public static class ToolRuleEvent extends Event {
+
+		private ToolRuleWrapper toolRuleWrapper;
+
+		public ToolRuleEvent(ToolRuleWrapper wrapper) {
+			toolRuleWrapper = wrapper;
+		}
+
+		public ToolRuleWrapper getToolRuleWrapper() {
+			return toolRuleWrapper;
+		}
+
+		@Override
+		public @NotNull HandlerList getHandlers() {
+			throw new IllegalStateException();
+		}
+	}
+
+	static {
+		Skript.registerSection(EffSecCreateToolRule.class, "create [a] [new] tool rule and store (it|the result) in %object%");
```

Comment:
Still not a fan of "store in" for this might be worth while to look into the ReturnHandler class to example
```applescript
create a new tool rule:
  return the tool rule
set {_toolrule} to last created tool rule
```